### DISCUSSION TITLE
Cannot change "Return to cart" text

### DISCUSSION THREAD

**Discussion by zippieq** [woocommerce/woocommerce#49009] [Checkout Flow]
Cannot change "Return to cart" text

**Comment by nielslange** [comment]
Hello @zippieq 👋

Currently, the "Return to cart" text cannot be changed within the editor. I created #49337 for taking care of this. For now, you could use the [Say What?](https://wordpress.org/plugins/say-what/) plugin for editing the text. Here are the steps you'd have to take:

1. Install and activate the Say What? plugin.
2. Go to `/wp-admin/tools.php?page=say_what_admin&say_what_action=addedit`.
3. Provide the following information:

- Original string: **Return to Cart**
- Text domain: **woocommerce**
- Text content: _you can leave this field empty_
- Replacement string: _you can provide your desired text_

The form should look like this:

<img width="1130" alt="Screenshot 2024-07-10 at 13 42 49" src="https://github.com/woocommerce/woocommerce/assets/3323310/94965d96-2f9b-4b4e-9595-101425e5dd30">

This then leads to the following result on the Checkout block:

<img width="837" alt="Screenshot 2024-07-10 at 13 43 22" src="https://github.com/woocommerce/woocommerce/assets/3323310/a7b460e8-22c1-4c86-8c76-9c097445ab7e">

As I've created #49337 and there's no action item left here, I'm closing this discussion. If you have further questions or suggestions, feel free to create an issue via https://github.com/woocommerce/woocommerce/issues.
